### PR TITLE
feat(frontend): add Snow effect

### PR DESCRIPTION
**Pull request by AntonioVentilii**
# Motivation

For the new year holidays - let's bring some snow to OISY users. 

# Changes

- Added the [svelte-confetti](https://mitcheljager.github.io/svelte-confetti/) library to add a snow on the background. 

# Tests

Only tested on the home page: 

https://github.com/user-attachments/assets/0f5165d9-2fc7-4610-ad93-33cdf622ddd3

**From Artem:** Unfortunately I can only test properly on on localhost without API keys, please re-test it on your end.

### CODE REVIEW

File: src/frontend/src/lib/components/ui/SnowBackground.svelte [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+	import Confetti from 'svelte-confetti';
+	import snow1 from '$lib/assets/snow-1.svg'
+	import snow2 from '$lib/assets/snow-2.svg'
+	import snow3 from '$lib/assets/snow-3.svg'
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/components/ui/SnowBackground.svelte:5]: in this folder you can change the assets

Comment (artkorotkikh-dfinity) [reply src/frontend/src/lib/components/ui/SnowBackground.svelte]: Thanks I updated assets

File: src/frontend/src/lib/components/ui/SnowBackground.svelte [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+	const colorArray = [
+		`url(${snow1})`,
+		`url(${snow2})`,
+		`url(${snow3})`,
+	];
```
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply src/frontend/src/lib/components/ui/SnowBackground.svelte]: We probably do not need the random, the animation already feels like being so.

**Comment by peterpeterparker** [comment]
Since @AntonioVentilii @artkorotkikh-dfinity and I worked on this PR, I assigned the review specially to @DenysKarmazynDFINITY

**Review by DenysKarmazynDFINITY — APPROVED** [review]
LGTM, thanks

**Pull request merged**